requirements-dev.txt

```diff
@@ -73,7 +73,7 @@ faker==25.2.0
     #   factory-boy
 filelock==3.14.0
     # via virtualenv
-flake8==7.0.0
+flake8==7.1.0
```

Comment:
Since flake8/black/isort are supposed to be kept in sync with pre-commit, it might be wise to pin the version in requirements-dev.in and leave a comment about keeping them in sync with pre-commit 🧐

Reply:
Sounds good. :+1: